src/components/Icon/assets/generic/search filled.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
same here

Reply:
resolved at https://github.com/Orfium/orfium-ictinus/pull/823/commits/a3879a6f77774c1d9d1257360457ec6bebbb5f79

Reply:
@panvourtsis dont forget to update snapshots because tests are broken atm

Reply:
fixed at https://github.com/Orfium/orfium-ictinus/pull/823/commits/9b6079d2978d744e61fe9d1282113fec22ecc4b6